circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceTraitCodec.scala

```diff
@@ -114,6 +105,33 @@ trait CirceTraitCodec[Ser <: AnyRef] extends Codec[Ser] {
     Json.obj((manifestString, encoder.asInstanceOf[Encoder[Ser]](a)))
   }
 
+  def manifest(o: AnyRef): String = parentsUpToRegisteredTypeMap(o.getClass.getName)
+
+  /*
+   * All code below serves as a check - it checks,
+   * whether class extending this trait is a valid implementation.
+   * doNeededChecksOnStart() gets invoked on object creation.
+   */
+  doNeededChecksOnStart()
+
+  private def doNeededChecksOnStart(): Unit = {
+    checkImplementationForInvalidMembersDeclarations()
+    checkSerializableTypesForMissingCodec(packagePrefix)
+    checkCodecsForNull()
+    checkCodecsForDuplication()
+  }
+
+  private def checkImplementationForInvalidMembersDeclarations(): Unit = {
```

Comment:
Nit:
```suggestion
  private def checkImplementationForInvalidMemberDeclarations(): Unit = {
```
https://virtuslab.github.io/tips/#english/grammar-tips-1 point 4